docs-internal/Continuous-deployment.md

```diff
@@ -0,0 +1,47 @@
+# Continuous Deployment
+
+Having CD configured in an automated way, allows us to:
+
+- make the release task less time-consuming and less prone to human errors
+- decrease the attack surface by not needing maintainers to have publish access
+
+## Setup
+
+### Travis CI
+
+The configuration file is located at `.travis.yml`.
+
+Requirements: this configuration assumes a `NPM_TOKEN` environment variable defined in Travis's
+settings. (see [travis docs about env vars][travis-docs-env-vars])
+
+Notes:
+
+- `before_deploy`
+  
+  This hook runs after the build and testing steps have finished, and is purpose is twofold:
+
+  1. Clean-up uncommitted changes resulted from installing or building, e.g.: lockfiles
```

Comment:
Technically the `git stash` probably isn't necessary, since npm ignores the `.gitignore` by default and it's unlikely travis will have changed any commits that could be tracked.